src/effects/builtin/whitenoiseeffect.cpp

```diff
@@ -35,7 +35,7 @@ EffectManifestPointer WhiteNoiseEffect::getManifest() {
 }
 
 WhiteNoiseEffect::WhiteNoiseEffect(EngineEffect* pEffect)
-        : m_pLoudnessParameter(pEffect->getParameterById("Loudness")) {
+        : m_pLoudnessParameter(pEffect->getParameterById("Dry/Wet")) {
```

Comment:
```suggestion
        : m_pDryWetParameter(pEffect->getParameterById("Dry/Wet")) {
```
Rename all the code variables too